packages/swap/src/index.ts

```diff
@@ -827,42 +819,13 @@ export class Swap {
     const signers: Account[] = [];
 
     // Add instruction to batch create all open orders accounts, if needed.
-    let accounts: Account[] = [];
-    if (fromNeedsOpenOrders || true) {
-      const oo = new Account();
-      signers.push(oo);
-      accounts.push(oo);
-    }
-    if (toNeedsOpenOrders || true) {
-      const oo = new Account();
-      signers.push(oo);
-      accounts.push(oo);
-    }
-    if (fromNeedsOpenOrders || toNeedsOpenOrders || true) {
-      let remainingAccounts = accounts.map((a) => {
-        return {
-          pubkey: a.publicKey,
-          isSigner: true,
-          isWritable: true,
-        };
-      });
-      const openOrdersSize = 200;
-      const lamports = new BN(
-        await this.program.provider.connection.getMinimumBalanceForRentExemption(
-          openOrdersSize,
-        ),
-      );
-      // ixs.push(
-      // 	this.createAccountsProgram.instruction.createAccounts({
-      // 		accounts: {
-      // 			funding: this.program.provider.wallet.publicKey,
-      // 			owner: DEX_PID,
-      // 			systemProgram: SystemProgram.programId,
-      // 		},
-      // 		remainingAccounts,
-      // 	}),
-      // );
-    }
+    const accounts: Account[] = [];
```

Comment:
Somewhat of non-lint change but fine, swap isn't used.

Reply:
Yeah, the linter complains that the following two if statements are always executed because of the `true`. Also we never reassign accounts to a different pointer, we just modify it, so it was complaining about that too.